api/src/main/java/jakarta/config/Config.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package jakarta.config;
+
+import java.util.Optional;
+
+import jakarta.config.spi.ConfigSource;
+import jakarta.config.spi.Converter;
+
+/**
+ * Access to configuration values.
+ */
+public interface Config {
```

Comment:
You can't get a Configuration value from this interface (yet)?

Reply:
Yes. That was still under discussion, so only the minimum things were brought up.

Reply:
The point is that it's completely useless right now because by definition a least common denominator also at least needs to work.